testsuite/cluster-test/src/cluster_builder.rs

```diff
@@ -17,7 +17,7 @@ use futures::{future::try_join_all, try_join};
 use libra_logger::info;
 use structopt::StructOpt;
 
-#[derive(StructOpt, Debug)]
+#[derive(Clone, StructOpt, Debug)]
```

Comment:
Btw, clone is not needed too right?

Reply:
I use clone so that we can give the ClusterTestRunner a clone of the params

Reply:
I see, got it